powerVCF.psm1

```diff
@@ -1497,6 +1556,119 @@ Export-ModuleMember -Function Set-VCFCredential
 
 ######### End Credential Operations ##########
 
+######### Start Credential Failed Task Cancel Operation ##########
+
+Function Cancel-VCFCredentialTask {
+
+<#
+    .SYNOPSIS
+    Connects to the specified SDDC Manager and cancels a failed update or rotate passwords task.
+
+    .DESCRIPTION
+	The Cancel-VCFCredentialTask cmdlet connects to the specified SDDC Manager and cancles a failed update or rotate passwords task.
+
+    .EXAMPLE
+	PS C:\> Cancel-VCFCredentialTask -id 4d661acc-2be6-491d-9256-ba3c78020e5d
+    This example shows how to cancel a failed rotate or update password task.
+#>
+
+	param (
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$id
+    )
+
+    if ($PsBoundParameters.ContainsKey("id")) {
+        $uri = "https://$sddcManager/v1/credentials/tasks/$id"
+    } else  {
+        Throw "task id to be cancelled is not provided"
+    }
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    try {
+        $response = Invoke-RestMethod -Method DELETE -URI $uri -ContentType application/json -headers $headers 
+        $response
+    }
+    catch {
+        #Get response from the exception
+        ResponseExeception
+    }
+
+}
+
+Export-ModuleMember -Function Cancel-VCFCredentialTask
+
+######### End Credential Failed Task Cancel Operation ##########
+
+######### Start Retry Credential Failed Task Rotate/Update operation ##########
+
+Function Retry-VCFCredentialTask {
+
+<#
+    .SYNOPSIS
+    Connects to the specified SDDC Manager and retry a failed rotate/update passwords task
+
+    .DESCRIPTION
+	The Retry-VCFCredentialTask cmdlet connects to the specified SDDC Manager and retry a failed rotate/update password task
+
+    .EXAMPLE
+	PS C:\> Retry-VCFCredentialTask -id -id 4d661acc-2be6-491d-9256-ba3c78020e5d -json .\Credential\updateCredentialSpec.json
+    This example shows how to update passwords of a resource type using a json spec
+
+    .EXAMPLE
+	PS C:\> Retry-VCFCredentialTask -id -id 4d661acc-2be6-491d-9256-ba3c78020e5d -json .\Credential\rotateCredentialSpec.json
+    This example shows how to rotate passwords of a resource type using a json spec
+#>
+
+	param (
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$privilegedUsername,
+        [Parameter (Mandatory=$true)]
+            [ValidateNotNullOrEmpty()]
+            [string]$privilegedPassword,
+        [Parameter (Mandatory=$true)]
```

Comment:
Asking for $privilegedUsername and $privilegedPassword inputs as required but examples are not showing the inputs

Reply:
Good catch. Fixed.